### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/markdown/code/vue-playground.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#839] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/markdown/code/vue-playground.html

提示 由于我们提供了一个运行时编译器，我们引入了带有 TypeScript 支持的整个 @vue/compiler-sfc 包，因此整个 Vue Playground 块大于 4MB。 因此，只有在严重依赖交互式 Vue Playground 时才应使用它。 你可以使用 和 作为替代。 配置 在你的项目中安装 @vue/repl: 之后启用它: 用法 ...

https://theme-hope.vuejs.press/zh/guide/markdown/code/vue-playground.html

**Comment by topoadmin** [comment]
幻灯片中能集成演示进去吗

**Reply by Mister-Hope** [reply]
不能，幻灯片的内容是由reveal.js库解析处理变成页面和动画，所以你只能使用它支持的功能。

所以说你既不能使用它不支持的扩展语法，也不能丢给它一个vue组件之类的东西。它都不懂。

这么解释理解么？

**Reply by topoadmin** [reply]
理解。

目前的解决方法是使用 iframe 嵌套到幻灯片中，也算是能达到需求了😂。

<iframe src="./demo.md" frameborder="no" border="0" ></iframe>

---

文件名：demo.md

::: vue-playground

@file App.vue

```vue
// 相关演示代码
```

:::
